### ISSUE TITLE
Accessibility enhancement 

### ISSUE THREAD

**Issue by tishoyanchev** [Infineon/infineon-design-system-stencil#1264]
- All components must meet the level AA of of [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG22/) 
- The aria- attributes of all components must be customizable so as to allow the user to define them in accordance with their application. 
- Only accessibility attributes that have no alternative can be hardcoded into the component. 
- Run the components through the [wave accessibility tool](https://wave.webaim.org/) to check if they are compatible with standards.

**Comment by tom-weisser-aracom** [comment]
I think that this is an important issue and i would like to see this implemented. Here are some observations i made, regarding keyboard control.

As far as i can see, some components, and therefore possibly entire pages, are not usable only via keyboard.
Some components are not focusable via tab. (e.g. Accordion, Navbar menu items)
Some components are focusable but not usable via keyboard. (e.g. Buttons and radio buttons cannot be pressed via enter or space)
